railties/lib/rails/generators/rails/app/templates/config/databases/sqlite3.yml.tt

```diff
@@ -8,6 +8,11 @@ default: &default
   adapter: sqlite3
   pool: <%%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
   timeout: 5000
+  journal_mode: WAL
+  synchronous: NORMAL
+  mmap_size: 134217728
+  journal_size_limit: 67108864
+  cache_size: 2000
```

Comment:
I think we should put back the comments that were originally in the `configure_connection` code to explain what these pragmas are doing and like to the SQLite docs.

Reply:
IMHO the comments don't add value.   Comments are code, in that they need to be maintained.  One seeking to understand what the pragma's do should seek out the source of truth which is the official documentation as the behavior may change.